frontend-new/src/pages/index.vue

```diff
@@ -1,12 +1,187 @@
 <script setup lang="ts">
-const { t } = useI18n();
+import {Menu, MenuButton, MenuItem, MenuItems} from "@headlessui/vue";
+
+const {t} = useI18n();
+
+const sorters = [
+    {id: 'stars', label: t("project.sorting.mostStars")},
+    {id: 'downloads', label: t("project.sorting.mostDownloads")},
+    {id: 'views', label: t("project.sorting.mostViews")},
+    {id: 'newest', label: t("project.sorting.newest")},
+    {id: 'updated', label: t("project.sorting.recentlyUpdated")},
+];
+
+const versions = [
```

Comment:
versions, categories, platforms and licences should be all loaded from backend eventually, doesn't need to be done now but we should have a TODO here